Makefile

```diff
@@ -266,16 +266,18 @@ run-memberagent: manifests generate fmt vet ## Run a controllers from your host.
 OUTPUT_TYPE ?= type=registry
 BUILDX_BUILDER_NAME ?= img-builder
 QEMU_VERSION ?= 7.2.0-1
+BUILDKIT_VERSION ?= 0.11.6-2
```

Comment:
0.11.6 released 2 years ago. what about `mcr.microsoft.com/oss/v2/moby/buildkit:v0.18.1` this one?